en/docs/release-notes/version-4.0.0.md

```diff
@@ -1,17 +1,74 @@
+!!! Compatibility
+    If you are new to Ballerina, you can download the [installers](https://ballerina.io/downloads/) to install it. You can install the Ballerina VS Code extension from the [VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina). To ensure all features work correctly, **install the latest patch release of Update 4 or later**.
+
+- [New unified visual and code editing experience](#new-unified-visual-and-code-editing-experience)
+- [HTTP API Designer](#http-api-designer)
+- [GraphQL API Designer](#graphql-api-designer)
+- [Architecture View](#architecture-view)
+- [Type Diagram](#type-diagram)
+- [Data Mapper improvements](#data-mapper-improvements)
+- [Language Server updates](#language-server-updates)
+- [Choreo extension integration](#choreo-extension-integration)
+
+## New unified visual and code editing experience
+
+In response to the valuable feedback received from developers, this release introduces a new unified development experience that seamlessly integrates code and visual development. This allows you to easily switch between code and visual development and leverage the benefits of both modes.
+
+With the new release, you can now find a **Visualize** CodeLens above each construct to visualize it. Clicking it will open the visualization of the construct. Currently, the visualization of all types of functions in Ballerina sequence diagrams and HTTP and GraphQL service types is supported. You can navigate back to the code using the menu or by pressing **Ctrl + Click** on the corresponding elements.
+
+## HTTP API Designer
+
+The extension now offers an HTTP Service Designer, which enables you to design HTTP services interactively. This feature allows you to design services rapidly even if you lack extensive knowledge of Ballerina's HTTP service syntax. To open the service designer, add an HTTP service using code or the **Add** button in the visual editor, and then click the **Visualize** CodeLens above the HTTP service.
+
+## GraphQL API Designer
+
+The GraphQL API Designer is introduced to streamline the visual creation of GraphQL APIs in Ballerina. This powerful tool simplifies understanding and navigating complex GraphQL services by letting you browse through them effortlessly without examining code directly. To open the GraphQL designer, add a GraphQL service from code using the service template and then, click the **Visualize** CodeLens above the service.
+
+## Architecture View
+
+The extension now includes a feature to visualize interactions and dependencies of multi-package Ballerina projects. Load all the packages into the VS Code workspace and select the **Architecture View** option to access it. The architecture diagram lets you view all the services in the packages and their interactions.
```

Comment:
I think `select the **Architecture View** option` has to be rephrased as `execute the **Architecture View** command`, since there is no visible option for this anywhere. The user has to specifically execute this from the VS Code command palette.

Reply:
Looping in @hevayo @kaviththiranga

Reply:
+1